openapi/platform/platform.yaml

```
@@ -2071,9 +2070,7 @@ paths:
       description: |-
         [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
 
-        Returns custom metadata about the sales account.
-
-        Attributes can be set up at the partner administration page in Partner Center.
+        This endpoint acts like a alias to [Get Sales Account Custom Data by ID endpoint](/platform/branches/np-ag-1371-reform-sales-account/b3A6MzYxMTM5MTY-get-sales-account-custom-data-by-id). It returns custom fields about a sales account.
```

The absolute link will break when we merge to master and delete this branch.

```suggestion
        This endpoint acts like a alias to [Get Sales Account Custom Data by ID endpoint](b3A6MzYxMTM5MTY-get-sales-account-custom-data-by-id). It returns custom fields about a sales account.
```

---

I have updated this.